api/source/basics.rst

```diff
@@ -99,232 +99,419 @@ Languages
+| Swahili                                  | sw                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Swedish                                  | se                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Tagalog (Filipino)                       | tl                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Tamil                                    | ta                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Telugu                                   | te                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Thai                                     | th                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Turkish                                  | tr                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Ukrainian                                | uk                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Urdu                                     | ur                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Uzbek                                    | uz                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Vietnamese                               | vn                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Xhosa                                    | xh                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+| Zulu                                     | zu                  | ✅                  | ✅                  |
++------------------------------------------+---------------------+---------------------+---------------------+
+
+
+
+Standard
```

Comment:
Move this table up to be after the HIGH fidelity table

Reply:
Done